source/docs/advanced/performance/cache-control-and-cdn.md

```diff
@@ -11,7 +12,7 @@ To enable CDN or reverse-proxy caching you first need to ensure that your site s
 
 In Front-Commerce we have implemented a mechanism where you can easily set up the `cache-control` headers for different parts of your site. You can also provide a custom implementation of the `cache-control` if needed (e.g. products with old last modified have longer caching duration).
 
-This page explains all the tools at your disposal to implement a totally custom, fine-grained caching strategy.
+Implement a totally custom, fine-grained caching strategy is a matter of compromises: freshness of data vs performance. It is **NOT** something that Front-Commerce can do by default because every context is important. We recommend that you spend time understanding this Front-Commerce feature and update your application accordingly.
```

Comment:
```suggestion
Implementing a totally custom, fine-grained caching strategy is a matter of compromise: the freshness of data vs performance. It is **NOT** something that Front-Commerce can do by default because every context is important. We recommend that you spend time understanding this Front-Commerce feature and update your application accordingly.
```